libdino/src/service/util.vala

```diff
@@ -26,6 +45,33 @@ public class Util {
         }
         assert_not_reached();
     }
-}
 
+    public static void launch_default_for_uri(string file_uri)
+    {
+#if _WIN32
+        ShellExecute("open", file_uri);
```

Comment:
Any reason this fallback is done in Dino instead of Glib?

Reply:
Even though this works fine for Dino, it doesn't necessarily gives the control that GLib wants. See: https://gitlab.gnome.org/GNOME/glib/-/issues/1932#note_649492

This was also asked here: https://gitlab.gnome.org/GNOME/glib/-/issues/2088
With no direct answer. I can try to contact them on IRC to understand the issue completely, if you want.